include/flatbuffers/flatbuffers.h

```diff
@@ -1344,9 +1344,9 @@ class FlatBufferBuilder {
   // Write a single aligned scalar to the buffer
   template<typename T> uoffset_t PushElement(T element) {
     AssertScalarT<T>();
-    T litle_endian_element = EndianScalar(element);
+    T little_endian_element = EndianScalar(element);
```

Comment:
Why this temporary is needed?
Can it be rewritten as:
```cpp
Align(sizeof(T));
buf_.push_small(EndianScalar(element));
```
?

Reply:
I was just fixing the spelling of the name, but I can rework it as well if you want.